dodo.py

```diff
@@ -57,21 +57,26 @@
 
 def envs(sep=' ', **kwargs):
     envs = dict(
-        DEPLOYED_ENV=CFG.DEPLOYED_ENV,
-        PROJECT_NAME=CFG.PROJECT_NAME,
         BRANCH=CFG.BRANCH,
-        REVISION=CFG.REVISION,
-        VERSION=CFG.VERSION,
-        REMOTE_ORIGIN_URL=CFG.REMOTE_ORIGIN_URL,
+        DEPLOYED_BY=CFG.DEPLOYED_BY,
+        DEPLOY_DOMAIN=CFG.DEPLOY_DOMAIN,
```

Comment:
ugh dont break up the DEPLOYED_ group of three please

Reply:
Sorry was just putting them in alpha order to be able to find the ones of interest.